internal/envoy/auth/filters.go

```diff
@@ -33,10 +35,8 @@ const (
 )
 
 func timeoutDefault() *durationpb.Duration {
-	return &durationpb.Duration{
-		Seconds: 16,
-	}
+	return durationpb.New(time.Second * 32)
```

Comment:
why change from 16 to 32?

Reply:
Didn't notice that
Good question. perhaps leave a comment with your intention @mbana why we need 32 seconds

Reply:
OK.

You want me to revert it? I just think a bigger timeout would be better.

Shout if you want me to revert it.

Reply:
I'm fine with it